korangar/src/world/entity/mod.rs

```diff
@@ -100,9 +102,9 @@ pub enum EntityType {
     Warp,
 }
 
-impl From<usize> for EntityType {
-    fn from(value: usize) -> Self {
-        match value {
+impl From<JobId> for EntityType {
+    fn from(job_id: JobId) -> Self {
+        match job_id.0 {
             45 => EntityType::Warp,
             111 => EntityType::Hidden, // TODO: check that this is correct
```

Comment:
Side note (not touched here): According to the [Hercules docs](https://github.com/HerculesWS/Hercules/blob/c1fa5502eb08143a0e6e7862577fa57480919c0f/doc/constants_pre-re.md?plain=1#L2527) this is indeed correct.